CMSIS/DSP/DSP_Lib_TestSuite/DspLibTest_SV_FVP/ARM/DspLibTest_FVP.uvprojx

```diff
@@ -425,7 +425,7 @@
                 <v6Rtti>2</v6Rtti>
                 <VariousControls>
                   <MiscControls></MiscControls>
-                  <Define>ARM_MATH_MATRIX_CHECK, ARM_MATH_ROUNDING, ARM_MATH_LOOPUNROLL</Define>
+                  <Define>ARM_MATH_MATRIX_CHECK, ARM_MATH_ROUNDING</Define>
```

Comment:
The change is applicable only for CMSIS-NN. ARM_METH_LOOPUNROLL is valid for CMSIS-DSP. Please remove all changes from CMSIS-DSP.

Reply:
Ok, it's done